### DISCUSSION TITLE
How to set the number of parallel CPUs for older versions of MOOSE

### DISCUSSION THREAD

**Discussion by Ating24** [idaholab/moose#26824] [Q&A Tools]
I used an older version of MOOSE on the cluster for sensitivity analysis (which had invalid min/max_procs_per-app values), and used a total of 80 CPUs to calculate 256 sample data. I want to calculate 2 samples at a time, with 40 CPUs per sample. Or calculate one sample at a time, one CPU per sample. How do I operate it?

**Comment by grmnptr** [comment]
Please update your moose version, the control over processor per app was introduced in 2021. If you don't have access to that, I am afraid we cannot help you.
